invenio_records_resources/services/custom_fields/base.py

```diff
@@ -36,6 +30,13 @@ def ui_field(self):
         """Marshmallow UI field for custom fields."""
         return self.field
 
+    @abstractmethod
+    def mapping(self, search_version=None):
```

Comment:
comment: default to `None` seemed better than defaulting to ES or OS. Then the default can be set at invenio-cli (according to what was chosen when bootstrapping the instance).